Modules/posixmodule.c

```diff
@@ -13886,7 +13902,7 @@ os_set_inheritable_impl(PyObject *module, int fd, int inheritable)
 }
 
 
-#ifdef MS_WINDOWS
+#if defined(MS_WINDOWS) && !defined(MS_WINDOWS_NON_DESKTOP)
```

Comment:
Why doesn't this work for the games API family? 

The SDK supports `GetHandleInformation()` and `SetHandleInformation()` for `WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP | WINAPI_PARTITION_SYSTEM | WINAPI_PARTITION_GAMES)`. However, "WinBase.h" defines the macro `HANDLE_FLAG_INHERIT` only for `WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP | WINAPI_PARTITION_GAMES)`. This excludes UWP store/phone apps, but it shouldn't exclude Xbox.

Reply:
The documentation states it is only available on Desktop. At least as far as I could see this is done by the SDK as well: https://github.com/tpn/winsdk-10/blob/9b69fd26ac0c7d0b83d378dba01080e93349c2ed/Include/10.0.10240.0/um/WinBase.h#L1810

Reply:
I'll have to be more careful about this. I checked SDK version 10.0.22621.0 (Windows 11) instead of 10.0.20348.0.

Reply:
I double checked this in 10.0.19041. `GetHandleInformation()` and `SetHandleInformation()` are available for the app, system, and games API partitions, i.e. they're always available. In 10.0.18362, they're limited to the desktop and system API partitions. Since we're assuming the 10.0.19041 SDK when defining the `mmap` module, the same assumption should apply here.

Reply:
Yes this API is available, but `HANDLE_FLAG_INHERIT` is only available on desktop. However I find this strange that `SetHandleInformation` is available without any of the flags.

Reply:
Can you check whether manually defining the flag allows `os.get_handle_inheritable()` to work on Xbox? It should.

```c
#ifndef HANDLE_FLAG_INHERIT
#define HANDLE_FLAG_INHERIT             0x00000001
@endif
```

Reply:
I can give it a try tomorrow.

Reply:
I tried:
```python
    try:
        print(os.get_inheritable(0))
        os.set_inheritable(0, 0)
        print(os.get_inheritable(0))
    except Exception as e:
        print(e)

    try:
        print(os.get_inheritable(0))
        os.set_inheritable(0, 1)
        print(os.get_inheritable(0))
    except Exception as e:
        print(e)
```
leads to access denied:
```
False
[WinError 5] Access is denied
False
[WinError 5] Access is denied
```

Which I guess is more true to the reality on the system.